src/Forms/NumericField.php

```diff
@@ -101,10 +101,24 @@ protected function getNumberType()
         return NumberFormatter::TYPE_DOUBLE;
     }
 
+    /**
+     * In some cases and locales, validation expects non-breaking spaces.
+     * This homogenises narrow and regular NBSPs to a regular space character
+     *
+     * @param string|null $value
+     * @return string
+     */
+    private function clean($value): string
```

Comment:
```suggestion
     */
    private function clean(?string $value): string
```
Just use typehints (and don't duplicate with PHPDocs) for new methods where we know the types it accepts.